error.go

```diff
@@ -4,27 +4,47 @@ import (
 	"strings"
 )
 
+// Error type returned when checking responses
+// that contain an error.
 type ResponseError struct {
+	// High level status of the response, api_schema.go
 	statusCode int
+	// Project specific error response, see project that uses it
+	errorCode int
+	// Human readable (but not beautified) error text
 	statusText string
```

Comment:
The tags are in the `Response` object. That is going over the wire.

If we want to allow for marshalling of this error type, we should add the tags, but I'm not sure that is needed.